plugins/modules/proxmox_kvm.py

```diff
@@ -525,6 +525,12 @@
       - Update of O(pool) is disabled. It needs an additional API endpoint not covered by this module.
     type: bool
     default: false
+  update_unsafe:
+    description:
+      - If V(true), does not enforce limitations on parameters O(net), O(virtio), O(ide), O(sata), and O(scsi).
```

Comment:
This is missing two affected parameters:
```suggestion
      - If V(true), does not enforce limitations on parameters O(net), O(virtio), O(ide), O(sata), O(scsi), O(efidisk0), and O(tpmstate0).
```